core-icon.html

```diff
@@ -167,7 +167,11 @@
         }
 
         this.iconset = this.computeIconset(this.icon);
-        this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        if (this.iconsetElement) {
+          this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        } else {
+          console.warn('iconset "' + this.iconset + '" not found!');
```

Comment:
[`console.warn`](https://developer.mozilla.org/en-US/docs/Web/API/Console/warn) is not a standard and should likely not go in a production element.